dev/integration_tests/external_ui/lib/main.dart

```diff
@@ -27,17 +27,17 @@ enum FrameState { initial, slow, afterSlow, fast, afterFast }
 
 class MyAppState extends State<MyApp> with SingleTickerProviderStateMixin {
   int _widgetBuilds = 0;
-  FrameState _state;
+  late FrameState _state;
```

Comment:
Instead of `late` maybe initialize it to `FrameState.initial`?
```suggestion
  FrameState _state = FrameState.initial;
```